stl/inc/chrono

```diff
@@ -1331,15 +1331,16 @@ namespace chrono {
         return *this;
     }
 
-    inline constexpr day _Last_day_table[] = {
-        day{31}, day{28}, day{31}, day{30}, day{31}, day{30}, day{31}, day{31}, day{30}, day{31}, day{30}, day{31}};
+    // To prevent UB by going out of bounds, four extra days with an invalid day are added.

```

Comment:
Would saying how it prevents that be necessary? The only usage of this array is _right there_ 7 lines down.

Reply:
I think this comment is fine. The code is a little strange, so it's okay to spend one line and avoid potential confusion.